numpyro/distributions/transforms.py

```diff
@@ -41,6 +41,7 @@
     "PermuteTransform",
     "PowerTransform",
     "SigmoidTransform",
+    "Simplex2OrderedTransform",
```

Comment:
Could you think of another name for this transform? In our forum, I use this name just to clarify the idea. For end-user, I think that `SimplexToOrderedTransform` might be a better name.

Reply:
SimplexToOrderedTransform sounds good. I'll reflect it in all deps.